README.md

```diff
@@ -48,30 +48,29 @@ If you already have a Kentico Cloud account and you want to connect the sample a
 
 To preview content in the sample application, follow these steps:
 
-1. In Kentico Cloud, choose Project settings from the app menu.
-2. Under Development, choose API keys.
-3. Copy your Project ID and Preview API key.
-4. Open the `src\Client.js` file in the sample application folder.
-5. Find the `projectId` and `previewApiKey` constants.
-6. Change the values of the constants using the Project ID and Preview API key you copied.
-7. Save the file.
+1.  In Kentico Cloud, choose Project settings from the app menu.
```

Comment:
Could you also change the lists to start with just one space to fulfill the default indentation there?

https://github.com/DavidAnson/markdownlint/blob/v0.15.0/doc/Rules.md#md030